benchmark/net/tcp-raw-c2s.js

```diff
@@ -58,71 +59,71 @@ function main({ dur, len, type }) {
   };
 
   client(type, len);
-}
-
 
-function fail(err, syscall) {
-  throw util._errnoException(err, syscall);
-}
-
-function client(type, len) {
-  var chunk;
-  switch (type) {
-    case 'buf':
-      chunk = Buffer.alloc(len, 'x');
-      break;
-    case 'utf':
-      chunk = 'ü'.repeat(len / 2);
-      break;
-    case 'asc':
-      chunk = 'x'.repeat(len);
-      break;
-    default:
-      throw new Error(`invalid type: ${type}`);
+  function fail(err, syscall) {
```

Comment:
AFAIK not all function have to be moved inside `main`, e.g., `fail` should be fine as is.

It should also be less churn if the arguments are just passed to the functions instead of moving the functions.